sdk/spring/scripts/replace_util.py

```diff
@@ -62,6 +62,17 @@
             VERSION_UPDATE_ITEMS: (
                 VersionUpdateItem('org.springframework.boot:spring-boot-starter-parent', '2.2.9.RELEASE'),
             )
+        },
+        'sdk/spring/azure-spring-cloud-test-eventhubs/pom.xml': {
+            VERSION_UPDATE_ITEMS: (
+                VersionUpdateItem('org.springframework.boot:spring-boot-starter-parent', '2.2.9.RELEASE'),
+                # VersionUpdateItem('org.springframework.cloud:spring-cloud-stream', '2.2.1.RELEASE'), # EventHubIT.integrationTest will throw ClassNotFoundException.
```

Comment:
FYI:

We should use `spring-cloud-stream-3.0.x` for `spring-boot-2.2.x`.

Refs: https://spring.io/projects/spring-cloud-stream

Reply:
cc: @saragluna , FYI.